src/OrbitDB.js

```diff
@@ -345,6 +345,11 @@ class OrbitDB {
     return Object.keys(databaseTypes).includes(type)
   }
 
+  static addType (type, store) {
```

Comment:
Not gonna bikeshed this, but what do you think about naming it `addDatabaseType`? If you're happy with the current name, then let's go with that.

If you do change it, make sure to reflect that in the documentation too.